packages/react/src/themes/teams-high-contrast/components/Dialog/dialogStyles.ts

```diff
@@ -0,0 +1,8 @@
+import { pxToRem } from '../../../../lib'
+import { ICSSInJSStyle } from '../../../types'
+
+export default {
+  root: ({ variables: v }): ICSSInJSStyle => ({
+    border: `${pxToRem(1)} solid ${v.foregroundColor}`,
```

Comment:
This is the second place throughout the whole theme where we use `pxToRem` for borders. Anywhere else we intentionally used `px` for border not to scale.
So what's the correct approach from design perspective?

Reply:
i think pixels were used for borders in hc theme originally because in some browsers the border will disappear when you scale down to 50%. but the more interesting question is if we care about support of scaling down that far?

cc/ @levithomason

Reply:
Chatted with @levithomason today about this topic. For this case (and most borders) we will want to use pixels since we don't want them to scale.